### PR TITLE
Test non-default IP/network in kubeadm

### PR DESCRIPTION
**Pull request by kashifest**
Updates the document with test Results of non-default IP/network experiment in kubeadm.

Jira Issue #AIR-146

### CODE REVIEW

File: airship/kubeadm-airship-gap-analysis.md [hunk 1]
```diff
@@ -25,6 +25,79 @@ Jira issues:
...
+## Test Cases
+* Apply kubeadm init and join with default configurations to check how it populates the different static pod manifest files
+* Use a custom kubeadm-init config file where we add only api-server related configurations and check if a different interface for api-server is assigned
+* Extend the 2nd test case by adding etcd-server related configurations in kubeadm-init config file and check if different interface for api-server and etcd-server gets assigned.
+* In all of the above cases, consider the impact of these configuration using kubeadm-join config file.
+
+## Test Results
+
+### Test case 1: etcd and api-server IP not given
+**Setup:** No custom kubeconfig-init nor kubeconfig-join provided
```
Comment (Xenwar) [inline airship/kubeadm-airship-gap-analysis.md]: All these are shown as one line in the markdown.
Please add a new line after each bold line

File: airship/kubeadm-airship-gap-analysis.md [hunk 2]
```diff
@@ -25,6 +25,79 @@ Jira issues:
 Jira issues:
 - [Non-default IP for control plane components](https://airship.atlassian.net/browse/AIR-146)
 
+## Motivation
+For this experiment our motivation is to know the answers to the following questions
```
Comment (Xenwar) [inline airship/kubeadm-airship-gap-analysis.md]: add the following at the end:
"regarding control plane components"

File: airship/kubeadm-airship-gap-analysis.md (hunk 1 above)
Comment (Xenwar) [reply airship/kubeadm-airship-gap-analysis.md]: the same goes with the other test cases

Comment (kashifest) [reply airship/kubeadm-airship-gap-analysis.md]: done

File: airship/kubeadm-airship-gap-analysis.md (hunk 2 above)
Comment (kashifest) [reply airship/kubeadm-airship-gap-analysis.md]: done

**Pull request merged**